templates/contacts/acc_active_email.html

```diff
@@ -0,0 +1,6 @@
+{% autoescape off %}
+Hi {{ user.username }},
+Please click on the link to confirm your registration,
+
+127.0.0.1:8000{% url 'activate' uidb64=uid token=token %}
```

Comment:
Please try to get the URL from django. not every server is 127.0.0.1

Reply:
done